specification/policyinsights/resource-manager/Microsoft.PolicyInsights/stable/2019-10-01/examples/Attestations_CreateResourceGroupScope.json

```diff
@@ -0,0 +1,90 @@
+{
+  "parameters": {
+    "subscriptionId": "35ee058e-5fa0-414c-8145-3ebb8d09b6e2",
+    "api-version": "2019-10-01",
+    "resourceGroupName": "myRg",
+    "attestationName": "790996e6-9871-4b1f-9cd9-ec42cd6ced1e",
```

Comment:
Question: Will these attestation get invalidated if for e.g. set of resources in the subscription changes?
I see expiredOn is that the only driver?

Reply:
An attestation is only for the resource it extends.  If it is sitting off a subscription it is just attesting that the subscription meets some requirement.  The only things that invalidates the state is expiresOn being hit, the user changing the state, or the subscription moving/changing tenant/etc... out of the assignment's scope.  Subscription changes will be handled in the background and attestations that are no longer valid will be cleaned up